src/main/java/com/aws/greengrass/clientdevices/auth/certificate/handlers/CertificateSubscriptionEventHandler.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.aws.greengrass.clientdevices.auth.metrics.handlers;
+package com.aws.greengrass.clientdevices.auth.certificate.handlers;
```

Comment:
Sorry I think I confused you. The handler should still be part of the metrics domain. This is the thing that knows about metrics.